deepchem/models/tensorgraph/tensor_graph.py

```diff
@@ -519,14 +524,17 @@ def _install_queue(self):
       for layer in self.features + self.labels + self.task_weights:
         layer.pre_queue = True
       return
+    inputs = self.features + self.labels + self.task_weights
+    if len(inputs) == 0:
+      return
```

Comment:
Is this just for submodules?  Do we have queue working for submodules?

Reply:
It came up in some of the test cases.  In practice it would obviously be pretty strange to have a model with no inputs, though that doesn't mean it could never happen.